### DISCUSSION TITLE
OpenAI Integration

### DISCUSSION THREAD

**Discussion by FlourishingHumanityCorporation** [chaskiq/chaskiq#967] [Ideas]
It would be extremely useful to have an OpenAI chatbot that has access to our company documents as the default chatbot for my Chaskiq. I'd expect that many people would find this extraordinary valuable :)

**Comment by michelson** [comment]
Hi!, we currently have a plugin for that. you can point to a vector database and the bot will answer accordingly, the plugin names is GPTQdrant.
